test/controllers/shifts_controller_test.rb

```diff
@@ -0,0 +1,48 @@
+require 'test_helper'
```

Comment:
These tests pass nicely - bravo!

Later we may move to using RSpec (or not!). We'll discuss it.